LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -3818,6 +3818,54 @@ private static void GetNuvotonConfigurationD(ISuperIO superIO, Manufacturer manu
 
                         break;
 
+                    case Model.ROG_STRIX_Z790_E_GAMING_WIFI: //NCT6798D
+
+                        // Volts
+                        v.Add(new Voltage("CPU Core Voltage", 0));
+                        v.Add(new Voltage("+5V", 1));
+                        v.Add(new Voltage("AV (Standby)", 2));
+                        v.Add(new Voltage("+3.3V", 3));
+                        v.Add(new Voltage("+12V", 4));
+                        v.Add(new Voltage("AUXTIN4 (VIN8)", 5));
+                        v.Add(new Voltage("AUXTIN0 (VIN4)", 6));
+                        v.Add(new Voltage("+3.3V (Standby)", 7));
+                        v.Add(new Voltage("CMOS Battery", 8));
+                        v.Add(new Voltage("VTT", 9));
+                        v.Add(new Voltage("Memory Controller", 10));
+                        v.Add(new Voltage("CPU L2 Cache", 11));
+                        v.Add(new Voltage("PCH 1.05V", 12));
+                        v.Add(new Voltage("CPU System Agent", 13));
+                        v.Add(new Voltage("CPU Input Voltage", 14));
+                        v.Add(new Voltage("SYSTIN (VIN9)", 15));
+
+                        // Temps
+                        t.Add(new Temperature("CPU Package", 0));
+                        t.Add(new Temperature("CPU", 1));
+                        t.Add(new Temperature("Motherboard", 2));
+                        t.Add(new Temperature("AUXTIN0", 3));
+                        t.Add(new Temperature("AUXTIN1", 4));
+                        t.Add(new Temperature("AUXTIN2", 5));
+                        t.Add(new Temperature("AUXTIN3", 6));
+                        t.Add(new Temperature("AUXTIN4", 7));
+
```

Comment:
Mind checking Temperature Index 13 and 22? Index 13 should be the Chipset, while 22 is another (and typically more responsive) COU temperature

A word on Auxiliary temperature sensors: Those tend to not actually include any meaningful values so I recommend removing those sensors. An obvious giveaway is a temperature never changing, or values going below ambient. Based off all your screenshots all should probably be removed.

Reply:
Agreed. Let me see what I can dig up.

Reply:
![image](https://github.com/user-attachments/assets/eaae8da8-bc31-43a2-b9f5-9b4205b3c15d)